app/controllers/posts_controller.rb

```diff
@@ -14,6 +14,28 @@ def show
     @likes_count = @post.likes.size
   end
 
+  def create
+    @post = current_user.posts.build(post_params)
+
+    if @post.save
+      redirect_to user_post_path(user_id: current_user.id, id: @post.id), notice: 'Post was successfully created.'
+    else
+      render :new
+    end
+  end
+
+  def new
+    @user = User.find(1) # Pronalazak korisnika sa ID-om 1
+    @post = @user.posts.build
+  end
+
+  def like
+    @post = Post.find(params[:id])
+    Like.create(post: @post, author: current_user)
+
+    respond_to(&:js)
```

Comment:
- [ ] Great job @djo1975, You did well adding the create method inside the  `CommentsController`. But I could not found your controller for the `LikesController`, Please create a controller for the `LikesController` class and update your routes accordingly. Please be reminded the likes controller should have a create action.